packages/react-native-web/src/vendor/react-native/VirtualizedList/index.js

```diff
@@ -733,19 +733,6 @@ class VirtualizedList extends React.PureComponent<Props, State> {
       }
     }
 
-    // For issue https://github.com/necolas/react-native-web/issues/995
-    this.invertedWheelEventHandler = (ev: any) => {
```

Comment:
This reverts a recent patch. What is the plan for handling inverted lists?

Reply:
Yikes, it looks like it would be easy to pull back in, so I'll give that a shot soon.  I'll add louder comments to it as well to help preserve in future merges.

Reply:
Alternatively we try to fix this properly using the approach here https://github.com/necolas/react-native-web/issues/2233#issuecomment-1050378002

Reply:
Ok, although I don't understand the utility of the `inverted` option or inverting the scroll wheel direction, I have local changes which I can see through localhost:3000/lists does look successfully restore the behavior introduced by https://github.com/necolas/react-native-web/pull/2223.  Here is a PR and commit which does so: https://github.com/DavidRieman/react-native-web/pull/1.

Unfortunately something weird happened to git's interpretation of the relationship between our repositories.  Github offered to merge necolas/react-native-web into davidrieman/react-native-web and I hesitated because I'd seen my PRs "Closed" (yet somehow merged) rather than "Merged" in GitHub - but ultimately I took it up on that auto-merge.  After that, however, I was unable to `git pull` locally anymore without error and had to re-clone my own repository locally and set everything back up.  Now I've committed these changes, but GitHub claims that the diff from my branch to your master is massive.  In these sorts of instances I find it safest to have a branch on the target repository cherry-pick the minimal commit to avoid whatever problems crept into place from the merge flow.  Would you want to cherry-pick 33e42909158de4c957c08b153474d654d6f84335 to your repo, or repeat the changes here?  Or should we handle another way?

Reply:
Oh wow, I thought all my PRs here were merged and in 17.7.  Just not **this** one.  Yeah I see now git will let me recover this branch and commit the fix.  So sorry for my misunderstanding, I'll try to have this patched up today.

Reply:
And cherry-pick worked.  Ready for review.

Reply:
I need an answer to this comment please https://github.com/necolas/react-native-web/pull/2241#issuecomment-1055824946

Reply:
Which issues are resolved?

Yes: The part of 1608 where I [linked to my SO](https://stackoverflow.com/questions/71152486/how-to-virtualize-a-flatlist-inside-flatlist-rn-web), and https://github.com/necolas/react-native-web/issues/2030

Yes partially (when combined with another upcoming potential patch, the behavior is improved, though still not perfect): https://github.com/necolas/react-native-web/issues/1798, https://github.com/necolas/react-native-web/issues/2026

Not Sure: https://github.com/necolas/react-native-web/issues/1769, https://github.com/necolas/react-native-web/issues/1798

Probably Not: https://github.com/necolas/react-native-web/issues/1292, https://github.com/necolas/react-native-web/issues/1579, https://github.com/necolas/react-native-web/issues/1608, https://github.com/necolas/react-native-web/issues/1873, https://github.com/necolas/react-native-web/issues/1880

No: https://github.com/necolas/react-native-web/issues/1254, https://github.com/necolas/react-native-web/issues/1854, https://github.com/necolas/react-native-web/issues/2233, https://github.com/necolas/react-native-web/issues/2239 (this one was filed as work that should follow later, but involves upgrading ScrollView itself which I didn't want to touch)

N/A: https://github.com/necolas/react-native-web/issues/2150 (still needs a full repro scenario IMO), https://github.com/necolas/react-native-web/issues/2167 (different issue, replied)